### DISCUSSION TITLE
Event for new plot, Multiple daily reports

### DISCUSSION THREAD

**Discussion by ricardoleal** [martomi/chiadog#105] [Ideas]
It would be great to have:

- Notification when a new plot finish;
- Possibility to have more than one Daily report.

Just some ideas ..

Thanks and great work so far !

**Comment by greimela** [comment]
Hi @ricardoleal, thanks for your feedback!

> Possibility to have more than one Daily report.

This is already planned, tracked in issue https://github.com/martomi/chiadog/issues/69.

> Notification when a new plot finish;

This is currently not planned, due to the reason stated in https://github.com/martomi/chiadog/issues/88:

> We're not going to have notifications for new plots because that's already included in the daily summary and receiving a notification for each individual plot is too much for most people.

**Reply by ricardoleal** [reply]
@greimela , thanks.

So, i guess we can approach in a different way. 

But. guess that can be tricky due the notifications alternatives.

In my case, i'm using telegram.
I can make my bot send a command and get the "status" ?
